apps/examples/testcase/Makefile

```diff
@@ -158,6 +158,9 @@ endif
 ifeq ($(CONFIG_EXAMPLES_TESTCASE_KERNEL),y)
 	$(Q) $(call REGISTER,kernel_tc,tc_kernel_main,TASH_EXECMD_ASYNC,180,2048)
 endif
+ifeq ($(CONFIG_EXAMPLES_TESTCASE_COMPRESSION),y)
```

Comment:
same as previous

Reply:
could you check here? This is not listed by Alphabet order.

Reply:
Done